week-1-ts-ecosystem/w01e01-typescript-basics/src/model/Address.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface AddressModel {
+    street: string,
+    no: number,
+}
```

Comment:
Address możesz schować razem z `User.model.ts` póki co - bo nigdzie nie jest "na zewnątrz używany".

Zastanów się też czy samo `*Model` do interface dodawać.
W nazwie pliku mogłoby zostać,
natomiast zakładając że masz to w folderze `model` to mogłoby być po prostu `User` 

Jednak to jedynie sugestia - do przemyślenia.